draft-ietf-quic-recovery.md

```diff
@@ -439,6 +456,14 @@ Either packet indicates that the Initial was received but not processed.
 Neither packet can be treated as an acknowledgment for the Initial, but they MAY
 be used to improve the RTT estimate.
 
+#### Discarding Initial State {#discard-initial}
+
+As described in Section 4.10 of {{QUIC-TLS}}, endpoints stop sending and
```

Comment:
I think it makes more sense to reference the transport draft, now that it has the required text.  Also, it's a bit odd to have the recovery draft have a direct dependency on TLS.

Most of this paragraph seems to apply to the Handshake encryption level as well, correct?